exercise/static/exercise/poll.js

```diff
@@ -63,7 +63,13 @@
 		},
 
 		ready: function() {
-			this.element.hide();
+			//this.element.hide();
+			
+			// For active elements the element to which the poll plugin is attached remains the same, so to
+			// be able to submit the same form several times the plugin data needs to be removed when the 
+			// evaluation and polling is finished.
+			if ($.data(this.element.context, "plugin_" + pluginName)) $.removeData(this.element.context, "plugin_" + pluginName);
```

Comment:
`.context` on jQuery objects is deprecated. Do you mean `this.element.get(0)` or is the context referring to the `document`? Originally, the data is set on the DOM element and since you are removing it, I assume that the document should not be used here.
https://api.jquery.com/context/
PS: jQuery has a higher level `.data` method too, so is it necessary to use the low level `$.data`? I know it was here previously too.